code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -348,14 +348,18 @@
 	metabolization_rate = 1
 
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
-	M.adjustBrainLoss(3)
 	if(iscultist(M))
 		M.status_flags |= GOTTAGOFAST
```

Comment:
Remove this, since this is an otherwise massive buff to the reagent, for cultists.

Don't forget to remove the `on_mob_delete` too.